packages/core/src/components/Pagination/index.tsx

```diff
@@ -143,7 +159,7 @@ export class Pagination extends React.Component<Props & WithStylesProps> {
           context="Showing the current page number and total page count"
         />
       ) : (
-        <T phrase={'%{pageNumber}'} pageNumber={page} context="Showing the current page number" />
+        <T phrase={page} context="Showing the current page number" />
```

Comment:
since `page` will be just a number with no formatting, no need for translation. update this to be just the `page` variable

Reply:
D'oh! Updated with changes.